mta/mta_services.go

```diff
@@ -357,21 +356,21 @@ func printResult(result interface{}, hashcode int, err error, print func(...inte
 	return err
 }
 
-// RunModifyAndWriteHash logs the info, executes the action while locking the mta file in the path, and writes the
-// result and hashcode (or error) to the output
+// RunModifyAndWriteHash - logs the info, executes the action while locking the MTA file in the path, and writes the
+// result and hashcode (or error, if needed) to the output.
 func RunModifyAndWriteHash(info string, path string, force bool, action func() error, hashcode int, isNew bool) error {
 	logs.Logger.Info(info)
 	newHashcode, err := ModifyMta(path, action, hashcode, force, isNew, os.MkdirAll)
 	writeErr := WriteResult(nil, newHashcode, err)
 	if err != nil {
-		// The original error is more important
+		// The original error has greater importance and will be displayed first.
```

Comment:
Please use this message instead:

If there is an error in both the “ModifyMta” function and the  “WriteResult” function, only the “ModifyMta” function returns the error.